workshops/kfp-caip-sklearn/lab-03-kfp-cicd/exercises/lab-03.ipynb

```diff
@@ -77,9 +77,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "%%writefile kfp-cli/Dockerfile\n",
```

Comment:
Per the lab exercise, this command is correct. The lab is intending to have the student write the Dockerfile. There is a Dockerfile in the kfp_cli folder but it is just a single line with a single TODO so this exercise is meant to overwrite that file as part of completion of the lab.

Reply:
Sorry for the confusion here. Since writing Dockerfile is a part of lab exercise here so closing this PR.